config.json

```diff
@@ -8,6 +8,7 @@
     "deviceName": "$hostname",
     "mac": "FF:FF:FF:FF:FF:FF",
     "port": 50045,
-    "pincode": "123-45-678"
+    "pincode": "123-45-678",
+    "listStoppedContainers":false
```

Comment:
For consistency it would be great, if you could add a space between the colon and the `false`!